scriptmodules/emulators/hatari.sh

```diff
@@ -13,6 +13,7 @@ rp_module_id="hatari"
 rp_module_desc="Atari emulator Hatari"
 rp_module_help="ROM Extensions: .st .stx .img .rom .raw .ipf .ctr .zip\n\nCopy your Atari ST games to $romdir/atarist\n\nCopy Atari ST BIOS (tos.img) to $biosdir"
 rp_module_licence="GPL2 https://hg.tuxfamily.org/mercurialroot/hatari/hatari/file/9ee1235233e9/gpl.txt"
```

Comment:
(Unrelated to the PR's functionality) License link is broken, https://git.tuxfamily.org/hatari/hatari.git/plain/gpl.txt should be ok.